extensions/wikia/TemplateClassification/TemplateClassification.hooks.php

```diff
@@ -98,12 +106,20 @@ public function onEditPageShowEditFormFields( \EditPage $editPage, \OutputPage $
 		global $wgCityId;
 
 		$context = \RequestContext::getMain();
+		$title = $context->getTitle();
 
-		if ( ( new Permissions() )->shouldDisplayEntryPoint( $context->getUser(), $context->getTitle() ) ) {
+		if ( ( new Permissions() )->shouldDisplayEntryPoint( $context->getUser(), $title ) ) {
 			$types = $this->getTemplateTypeForEdit( $editPage->getTitle(), $wgCityId );
```

Comment:
Because $context-getUser() it's used only once, while $context->getTitle() would have two invocations in this part of code